src/pms/cli.py

```diff
@@ -10,20 +10,24 @@
     import importlib_metadata as metadata
 
 from loguru import logger
-from typer import Argument, Context, Exit, Option, Typer, echo
+from typer import Argument, Context, Exit, Option, Typer, echo, run
 
 from pms.core import MessageReader, SensorReader, Supported, exit_on_fail
 
-main = Typer(help="Data acquisition and logging for Air Quality Sensors with UART interface")
-
+app = Typer(help="Data acquisition and logging for Air Quality Sensors with UART interface")
```

Comment:
why change the name of the CLI entry point?

Reply:
typer looks for an entry point called `main`, but I want to make this a function so I can override the logging behaviour before the main CLI starts. We can't have:

```
main = Typer(...)

...

def main(): ...
```